spec/index.bs

```diff
@@ -4682,6 +4695,12 @@ enum GPUTextureAspect {
                                 |this|.{{GPUTexture/format}},
                                 |descriptor|.{{GPUTextureViewDescriptor/aspect}}).
 
+                        - |descriptor|.{{GPUTextureViewDescriptor/usage}} must be a subset of |this|.{{GPUTexture/usage}}.
+                        - If |descriptor|.{{GPUTextureViewDescriptor/usage}} includes the {{GPUTextureUsage/RENDER_ATTACHMENT}} bit:
+                            - |descriptor|.{{GPUTextureViewDescriptor/format}} must be a [=renderable format=].
+                        - If |descriptor|.{{GPUTextureViewDescriptor/usage}} includes the {{GPUTextureUsage/STORAGE_BINDING}} bit:
+                            - |descriptor|.{{GPUTextureViewDescriptor/format}} must be listed in [[#plain-color-formats]] table
+                                with {{GPUTextureUsage/STORAGE_BINDING}} capability for at least one access mode.
                         - |descriptor|.{{GPUTextureViewDescriptor/mipLevelCount}} must be &gt; 0.
                         - |descriptor|.{{GPUTextureViewDescriptor/baseMipLevel}} +
                             |descriptor|.{{GPUTextureViewDescriptor/mipLevelCount}} must be &le;
```

Comment:
nit: The `[[renderExtent]]` thing below should probably look at the view usages, not the texture usages. Though it doesn't technically matter, we could honestly just set `[[renderExtent]]` unconditionally or make it a getter algorithm.

(found by looking through all usages of GPUTexture/usage to make sure there weren't any left that needed to be changed)